cloud/models/apps/insights/basic.go

```diff
@@ -0,0 +1,54 @@
+package insights
```

Comment:
The idea behind this is to separate the models from the request module to track versions, in this case its a little bit harder because the versions are 6-8 atm.
But for the other cases it would be:
- cloud/models/v2
- cloud/models/v3
- cloud/models/v4
- onpremise/models/v2
- .....